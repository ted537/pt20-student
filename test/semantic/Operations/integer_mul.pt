using output;
var out = 5 * 6;
