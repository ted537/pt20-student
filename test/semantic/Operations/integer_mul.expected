   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 2
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
   .tInitialValue
     .tLiteralInteger
     oEmitValue
     % value emitted 5
     .tLiteralInteger
     oEmitValue
     % value emitted 6
     .tMultiply
   .tInitEnd
   .tLiteralAddress
    .tStoreInteger
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
